content/index.html

```diff
@@ -52,7 +52,7 @@
 		<!-- End Google Analytics -->
 
 	</head>
-	<body style="height:100%">
+	<body ng-controller="navController" ng-style="isPublicProfile('/view') === true ? {'background': '#f5f8fa'} : {'background': 'white'}">
 		<nav class="navbar navbar-default" ng-controller="navController">
```

Comment:
You can remove the `ng-controller="navController"` from this line since you moved it up.